README.md

```diff
@@ -167,9 +163,7 @@ If you like this project...
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+I would like to thank sakwadesignstudio the author of the original design
 
```

Comment:
**[Mandatory]**

- [ ] Great job so far but you didn't do it correctly because you are just saying that **I would like to thank sakwadesignstudio** but no one knows who this person is and where to find him so should have made his name a link to his Behance account so whenever someone wants to verify then he could go to his account :blush:

:red_circle: Your Acknowledgment

```
I would like to thank sakwadesignstudio the author of the original design
```
:green_circle: Professional Acknowledgement

```
I would like to thank [sakwadesignstudio](https://www.behance.net/sakwadesignstudio) the author of the original [design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))
```